packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -1241,6 +1241,50 @@ export class Return extends Op {
   }
 }
 
+// push field from current transaction to stack
+export class Global extends Op {
+  readonly field: string;
+  readonly interpreter: Interpreter;
+
+  /**
+   * @param args words list extracted from line: {global field (string)}
+   * @param line line number in TEAL file
+   * @param interpreter interpreter object
+   */
+  constructor (args: string[], line: number, interpreter: Interpreter) {
+    super();
+    assertLen(args.length, 1, line);
+    this.assertGlobalDefined(args[0]);
+
+    this.field = args[0]; // field
+    this.interpreter = interpreter;
+  }
+
+  execute (stack: TEALStack): void {
+    let g;
```

Comment:
can we rename this to `result` or something more descriptive?

Reply:
renaming to `globalField`